ffi/wgpu.h

```diff
@@ -655,19 +693,6 @@ typedef struct {
   bool alpha_to_coverage_enabled;
 } WGPURenderPipelineDescriptor;
 
-typedef struct {
```

Comment:
now we need to declare this struct here, separately
the old struct turned to be Rust-only

Reply:
This is declared properly in #34 as it needs the pnext chain following. PR is fixing a wgpu PR before and after the one that #34 fixes, so we're kinda in this weird cyclical dependency state. I would say leave this as is for now until we can get #34 merged in right after this.

Reply:
ok, we can do that :)